app/services/publish_manual_service.rb

```diff
@@ -8,6 +10,8 @@ def initialize(dependencies)
 
   def call
     if versions_match?
+      tags
```

Comment:
I'd still be interested to hear your answer to my questions that this doesn't address, too (https://github.com/alphagov/specialist-publisher/pull/532#discussion-diff-36097376)